CoreAttributeEditorView/src/au/gov/asd/tac/constellation/views/attributeeditor/AttributeEditorPanel.java

```diff
@@ -675,6 +676,10 @@ private void createMultiValuePane(final AttributeData attribute, final TitledPan
             }
         });
         attributePane.setContent(dataAndMoreButtonBox);
+
+        //attributePane TitledPane EXPAND and COLLAPSE events
+        attributePane.expandedProperty().addListener((observable, wasExpanded, isNowExpanded)
```

Comment:
I'm not sure if there is a way to remove the listener - I couldn't find a place to do it in this class.